pkgs/tools/admin/bom/default.nix

```diff
@@ -6,7 +6,7 @@
 
 buildGoModule rec {
   pname = "bom";
-  version = "0.4.1";
+  version = "0.5.1";
```

Comment:
No hash change for an update? That's strange.

Reply:
I wondered too, but the bom did the update.

Reply:
While this does not seem to be true upstream: https://github.com/kubernetes-sigs/bom/compare/v0.4.1...v0.5.1

Reply:
I think the new hash should be `sha256-/a+0kQn2m5JFxhlgs1pUeyWwwqVFrRbYCyUfcLsNGz8=`.

Reply:
I used to use `nix hash path` to calculate the sha256, what is your method to do that?

Reply:
I simply remove the old hash and `nix-build`.